### ISSUE TITLE
[FIX] Global Crypto Stat Cards Blurry / Positioning off when resizing

### ISSUE THREAD

**Issue by eddybank** [arnab2001/Coinverse#80]
Similar to crypto/news cards when having a blur affect

**Comment by eddybank** [comment]
I would like this to be assigned to me.

**Comment by deepbeatz** [comment]
show a screenshot or screen recording plz

**Comment by eddybank** [comment]
Site that is live: 
![image](https://user-images.githubusercontent.com/89403905/223643914-8ec00825-6085-479c-ad84-fdea1508dafe.png)


My changes with the blur effect on child div rather than parent div.
![image](https://user-images.githubusercontent.com/89403905/223644086-0770be2a-18b5-476f-99ca-2aa8652ee05b.png)

**Comment by deepbeatz** [comment]
Ok understood, what changes did u do?

**Comment by eddybank** [comment]
![image](https://user-images.githubusercontent.com/89403905/223649091-867088f3-b285-4987-bdc6-94ac651602a6.png)

I swapped the 'ag-course_item' to use grid. 
![image](https://user-images.githubusercontent.com/89403905/223649713-0b177ca4-e256-4ed9-ac47-3dd0843cdca8.png)

Then placed the 'a' element onto the grid with a transparent background (ag-courses-item_link).
![image](https://user-images.githubusercontent.com/89403905/223649632-b85809a6-c2c7-4f38-bb98-61ba05112ed7.png)

I took the previous 'ag-courses-item_link' class and changed the name to 'ag-courses-item_bg-container' and added grid constraints. 
![image](https://user-images.githubusercontent.com/89403905/223649679-52a11dac-8d9d-4c5a-876a-0de4f34ce678.png)

And put the background div into the bg-container, so that only the background would be affected by the blur.
